test/operators.jl

```diff
@@ -58,6 +58,25 @@ p = 1=>:foo
 @test_throws ArgumentError Base.scalarmax('a',['c','d'])
 @test_throws ArgumentError Base.scalarmax(['a','b'],'c')
 
+s1 = Set([1])
+s2 = Set([2])
+@test_throws MethodError min(s1, s2)
+@test_throws MethodError max(s1, s2)
+@test_throws MethodError minmax(s1, s2)
+
+struct TO
+  x
+end
+Base.isless(a::TO, b::TO) = isless(a.x, b.x)
+Base.isequal(a::TO, b::TO) = isequal(a.x, b.x)
+
+@test min(TO(1), TO(2)) == TO(1)
```

Comment:
The test verify, that `min` and `max` work as expected, if the type `TO` defines a total ordering (by defining `isless`. Remember, that in the initial bug, they did not. Maybe I should have called `isequal` instead of `==`, which falls back to `===`. So actually I am testing an unessential implementation detail.